docs/guidelines/content-guidelines/02-style-and-terminology.md

```diff
@@ -128,41 +130,58 @@ Use short command line arguments whenever possible.
-
-- Config Map
-- Cron Job
-- Custom Resource Definition
-- Deployment
-- Function
-- Ingress
-- Namespace
-- Node
-- Pod Preset
-- Pod
-- Prow Job
-- Secret
-- Service
-- Service Binding
-- Service Class
-- Service Instance
-
-All Kyma components, such as Application Connector or API Gateway Controller, are capitalized as well. Whenever you point to the outside sources, do some research to check whether the name of the source starts with a capital letter or not.
-
->**NOTE:** Kubernetes itself is also capitalized. Do not write it in lowercase or in the abbreviated version.   
-  ✅ Kubernetes  
-  ⛔️ kubernetes   
-  ⛔️ k8s
+## Capitalization
+
+### Sentence case
+
+Sentence case is the normal style in English: Lowercase for all words, first letter of the first word in a sentence is uppercase. Exception: Uppercase for the first letters of proper nouns and names.
+
+Use sentence case for normal, everyday words in body text. Don't capitalize words just because they are "special" or "important".
+Also, we use use sentence case for headings (more details about [headings](03-formatting.md#headings)).
+
+### Upper case
 
-### Custom resources
+Upper case means the first letter of a word is capitalized.
 
-Use the natural language by default when writing about custom resources.
+Use upper case for Kyma components, such as Application Connector or API Gateway Controller.
+Whenever you point to the outside sources, research whether the name of the source starts with a capital letter or not.
 
-✅ API Gateway is a Kubernetes controller, which operates on API Rule custom resources.   
-⛔️ API Gateway is a Kubernetes controller, which operates on APIRule custom resources.
+### Camel case
 
-Use the CamelCase-spelled custom resource name if you want to refer to the code specifically. In that case, format it as code, for example, `APIRule`.
+Camel case means writing words or phrases without a blank space, using a capitalized letter to separate the words. It's very common in Kubernetes terminology.
+
+Use Camel case for Kubernetes resources and custom resources. For plurals, add "s".
+For navigation and titles, use natural language (upper case and blank space).
+
+✅ API Gateway is a Kubernetes controller, which operates on APIRule custom resources.  
+⛔️ API Gateway is a Kubernetes controller, which operates on API Rule custom resources.
+  
+If you refer to the code specifically, format it as code, for example, `APIRule` (more details about [code font](03-formatting.md#code-font-and-bold-font)).
+  
+However, if the words are not used in relation to Kubernetes resources, do not capitalize them.
+
+See the following examples:
+
+* ConfigMap
+* CronJob
+* CustomResourceDefinition (CRD) - note that "custom resource" alone isn't Kubernetes-specific, thus it's lower case.
```

Comment:
```suggestion
* CustomResourceDefinition (CRD) - note that "custom resource" alone isn't Kubernetes-specific, thus it's lowercase.
```
In AmE, it's usually written as one word.
See the [Cambridge dictionary](https://dictionary.cambridge.org/dictionary/english/lowercase) for reference.

Reply:
Thank you for spotting this!